monai/engines/multi_gpu_supervised_trainer.py

```diff
@@ -37,7 +37,7 @@ def _default_eval_transform(x, y, y_pred):
 def create_multigpu_supervised_trainer(
     net: torch.nn.Module,
     optimizer: Optimizer,
-    loss_fn: Callable,
+    loss_fn: torch.nn.modules.loss._Loss,
```

Comment:
I would suggest changing this back to `Callable`. There's no need for the loss function to be a Module type at all, it could be any callable which returns a tensor with gradients, that's all that's needed. I've seen loss functions implemented as Python routines before, I'm sure there's other loss functions implemented as direct subclasses of `nn.Module` too.

Reply:
Apologies if this has already been discussed, it's something that caught my attention here.

Reply:
My bad, I had my focus too much on torch losses; good to know.